lib/joken/current_time.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Joken.CurrentTime.OS do
```

Comment:
Can we get rid of this now and use Elixir's built in functions? https://hexdocs.pm/elixir/DateTime.html#to_unix/2. Only they are equivalent

Reply:
Done!